cirq-google/cirq_google/workflow/quantum_runtime.py

```diff
@@ -80,10 +81,13 @@ class RuntimeInfo:
             `cg.QuantumExecutable` was executed.
         qubit_placement: If a QubitPlacer was used, a record of the mapping
             from problem-qubits to device-qubits.
+        timings_s: A dictionary of the duration of various steps in the workflow,
+            measured in fractional seconds.
```

Comment:
Nit: this docstring (and ones above it too) seems like it is more ambiguous than it needs to be. Thoughts on: `timings: Dictionary mapping from str to float. Each entry corresponds to the number of seconds a particular execution subroutine took. <name all subroutines that are tracked>`

Mentioning things are timed in fractional seconds made me think "what denomination is used here do I have 40 half seconds which is really 20 full seconds etc. ?"

Reply:
I'm going to punt on listing all the possible names, as 1) it will likely change in the next couple of PRs 2) will depend on the particular `QuantumRuntimeConfiguration`, and I would like to document it there so all docstrings can point to one place.